exasol_sagemaker_extension/lua/src/autopilot_training_main.lua

```diff
@@ -155,6 +155,34 @@ function train_autopilot_caller(exa, args)
 	return job_name
 end
 
+---
+-- This method returns names and types of columns as comma-separated strings respectively.
+--
+-- @param schema_name		The name of schema where the table is in.
+-- @param table_name		The name of table from which column information is retrieved.
+--
+-- return two strings including column names and types
+--
+function get_table_columns(schema_name, table_name)
+	local query = [[SELECT COLUMN_NAME , COLUMN_TYPE FROM SYS.EXA_ALL_COLUMNS eac
+					WHERE COLUMN_SCHEMA = :schema_name AND COLUMN_TABLE = :table_name]]
+	local params = {schema_name=schema_name, table_name=table_name}
+
+	local success, res = _G.global_env.pquery(query, params)
+	if not success then
+		local error_obj = exaerror.create("",
+				"Error while getting columns information from SYS.EXA_ALL_COLUMNS: " ..  res.error_message)
+		_G.global_env.error(tostring(error_obj))
+	end
+
+   	local col_names, col_types = {}, {}
+	for i=1, #res do
```

Comment:
I am thinking, about adding a note about the size limitation to the user guide. And, if this should get a problem, we might need to create a table instead.

Reply:
Alternative question, do we get the columns from the trained model?

Reply:
- Getting column names from trained is not possible, since automl's training api does onl get values, not keeps columns names.
- We can extract the column names with the data to s3 and keep them there, but that would not be a very clean solution (because we will clean the s3 when the train is completed, but these columns names must remain. Moreover, keeping  metadata in both database and s3 will be redundant ..)

we can add a note about that limitation to user guide